### DISCUSSION TITLE
Email Address Conflict

### DISCUSSION THREAD

**Discussion by Mohiit70** [community/community#65355] [GitHub Education]
### Hi everyone,

Hey GitHub community,

I hope you are all doing well. I am facing an issue related to the GitHub Developer Pack and need some assistance.

I have two GitHub accounts, "MohitBisht700" and "Mohiit70". My main account is "Mohiit70", and I would like to avail the GitHub Developer Pack benefits with this account.

The problem is that I tried adding my college email address to my main account "Mohiit70," but I received an error stating that the email is already in use. It seems that the college email is currently connected to my other account "MohitBisht700," which I don't intend to use for the GitHub Developer Pack.

Could someone please guide me on how to resolve this email address conflict and use the GitHub Developer Pack benefits with my primary account "Mohiit70"? Is there a way to disconnect the college email from "MohitBisht700," or can I use a different email address (not connected to any GitHub account) to avail of the benefits?

I appreciate any help or insights you can provide to help me streamline my access to the GitHub Developer Pack resources.

Thank you all for your time and support.

Best regards,
Mohit Bisht

**Comment by samus-aran** [comment]
Hi @Mohiit70  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

**Reply by Mohiit70** [reply]
I would like to know if it's possible to delete the account where I initially activated the student pack. My intention is to remove my college ID from that account and then link it with a different account. I understand that I might need to unlink the college ID from the first account in order to associate it with my current account, allowing me to access the student pack.

**Comment by arafajs** [comment]
In your case, you have two GitHub accounts, "MohitBisht700" and "Mohiit70". Your main account is "Mohiit70", and you want to use the GitHub Developer Pack benefits with this account. However, the college email address that you want to use is currently connected to your other account "MohitBisht700".

To resolve this email address conflict, you can follow these steps:

1. Log in to your "MohitBisht700" account.
2. Go to your account settings.
3. Update the email address associated with that account to a different one that you don't intend to use for availing the GitHub Developer Pack benefits.
4. Once the college email is disassociated from "MohitBisht700", you should be now able to add it to your primary account "Mohiit70" without any issues.
5. Go to the settings of the "Mohiit70" account.
6. Add the college email address there.

**Reply by Mohiit70** [reply]
![git](https://github.com/community/community/assets/120316966/be03fae6-14ac-45b5-8c9b-56404c6fe7f9)

**Comment by Mohiit70** [comment]
> In your case, you have two GitHub accounts, "MohitBisht700" and "Mohiit70". Your main account is "Mohiit70", and you want to use the GitHub Developer Pack benefits with this account. However, the college email address that you want to use is currently connected to your other account "MohitBisht700".
> 
> To resolve this email address conflict, you can follow these steps:
> 
> 1. Log in to your "MohitBisht700" account.
> 2. Go to your account settings.
> 3. Update the email address associated with that account to a different one that you don't intend to use for availing the GitHub Developer Pack benefits.
> 4. Once the college email is disassociated from "MohitBisht700", you should be now able to add it to your primary account "Mohiit70" without any issues.
> 5. Go to the settings of the "Mohiit70" account.
> 6. Add the college email address there.

I already attempted this, but I'm encountering an error message.
